From 693f96fffde55e2a8dcbacd826ab42d4984ce58d Mon Sep 17 00:00:00 2001
From: tista500 <tista.gma500@gmail.com>
Date: Tue, 5 Mar 2019 13:49:49 +0900
Subject: [PATCH 1/1] Gtk-3.2x&4.0: [Budgie-desktop] Enforce clearing
 background color for MenuApplet categories pane

I didn't notice that solid background... :(
---
 src/gtk/sass/common/_3rd-party.scss | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gtk/sass/common/_3rd-party.scss b/src/gtk/sass/common/_3rd-party.scss
index 10b49de..08cb18d 100644
--- a/src/gtk/sass/common/_3rd-party.scss
+++ b/src/gtk/sass/common/_3rd-party.scss
@@ -492,6 +492,7 @@ button.flat.budgie-menu-launcher {
   }
 
   .categories { // left-paned elements of GtkPopover
+    background-color: transparent;
     &:dir(ltr) {
       margin-right: $popover_padding;
       border-right: 1px solid if($variant == dark, track(white, $a_div_2),
-- 
2.21.0

